src/components/Settings/SettingsAbout/Releases/index.tsx

```diff
@@ -18,13 +18,14 @@ const messages = defineMessages({
   latestversion: 'Latest',
   currentversion: 'Current Version',
   viewchangelog: 'View Changelog',
-  runningDevelop: 'You are running a develop version of Overseerr!',
+  runningDevelop: 'Development Version',
   runningDevelopMessage:
-    'The changes in your version will not be available below. Please see the <GithubLink>GitHub repository</GithubLink> for latest updates.',
+    'The latest changes to the <code>develop</code> branch of Overseerr are not shown below.\
+    Please see the commit history for this branch on <GithubLink>GitHub</GithubLink> for details.',
 });
 
 const REPO_RELEASE_API =
-  'https://api.github.com/repos/sct/overseerr/releases?per_page=20';
+  'https://api.github.com/repos/sct/overseerr/releases?per_page=10';
```

Comment:
Why 10?

Reply:
I looked at how often we release and didn't think it was useful to display that many.  The page looks cleaner and less intimidating with a more reasonable number of entries IMO.  There are currently only 19 entries _total_, so we aren't even hitting that limit of 20, and I think that just covers a really long period of time that wouldn't be useful to the user... so much has changed in that time!